src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -1963,6 +1965,35 @@ int TableDockWidget::getLabeledGroupCount()
   return _labeledGroups;
 }
 
+QString TableDockWidget::getTitleForId(int tableId)
+{
+  return _idTitleMap.value(tableId, QString(""));
+}
+
+bool TableDockWidget::setTitleForId(int tableId, const QString& tableTitle)
+{
+  if (_idTitleMap.contains(tableId))
```

Comment:
This is not being utilized during creation of the table.

Reply:
Can you clarify a little here? I don't understand what is not being used while creation of tables. The map?

Reply:
This checkpoint. If the tableId already exists and the function returns false, how are we using that information?

Reply:
Right, we are not using it in anyway in the constructors, because its unlikely that a newly created ID in the table will have an entry in the table ID map. However, a user can later call this method to check if it has been registered with a title. Let me know if I should remove this as well, if you don't see it being used anytime.

Reply:
I feel like if we are adding a method for error handling, we might as well actually handle that condition. However unlikely it might be. Not a major issue though. Take a call.